### PR TITLE
Fix: #13 - changed function that checks for sort column

### PR DESCRIPTION
**Pull request by yeahme49**

### CODE REVIEW

File: resources/lib/functions.py [hunk 1]
```diff
@@ -83,12 +83,19 @@ def build_url(query):
 	return base_url + '?' + urllib.parse.urlencode(query)
 
 def check_sort_db():
+#	cur = db.cursor()
```
Comment (mintsoft) [inline resources/lib/functions.py:86]: Can you delete these comments?

**Review by mintsoft — CHANGES_REQUESTED** [review]

File: resources/lib/functions.py (hunk 1 above)
Comment (yeahme49) [reply resources/lib/functions.py]: fixed

**Comment by mintsoft** [comment]
Cool, thanks @yeahme49

**Comment by yeahme49** [comment]
i haven't seen the update hit the kodi repo, just wanted to make sure you were going to submit it

**Comment by mintsoft** [comment]
@yeahme49 Yes, sorry. Been hectic, I'm packaging it up now

**Comment by mintsoft** [comment]
https://github.com/xbmc/repo-plugins/pull/3167 and https://github.com/xbmc/repo-plugins/pull/3166

**Pull request merged**